content/dogepedia/articles/get-dogecoin.md

```diff
@@ -5,4 +5,24 @@ date = "2021-10-22"
   name = "Dogecoin"
 +++
  
-Several ways to get your hands on Doge - from exchanges, tipping, or from projects like Dogecoin Folding @ Home. This could be broken down in subarticles, if needed.
\ No newline at end of file
+There are many ways to get your hands on Dogecoin: from exchanges, receiving tips, accepting Dogecoin for goods and services, and even from projects like Dogecoin Folding @ Home!  
+
+### Exchanges 
+
+A popular way people get their hands on Dogecoin is via an exchange. There are many exchanges on the market, and they are most commonly used to exchange fiat money for cryptocurrency, or to exchange cryptocurrency for fiat money.  
+
+You can purchase Dogecoin on an exchange at the current market price (this is called a market *taker* transaction), or by making an offer to purchase cryptocurrency once it reaches a specified price (a market *maker* transaction). Some exchanges only allow its users to purchase cryptocurrencies via market taker transactions.  
+
+Although the security of exchanges has come a long way in recent years, it is still crucial that you **do your own research (DYOR)** before purchasing Dogecoin via an exchange, because they are not immune to security risks or controversy. For added security, it is also important to consider keeping the Dogecoin that you do not intend to use for trading in a [private non-custodial wallet](https://dogecoin.com/dogepedia/articles/how-do-i-get-a-wallet).
+ 
+### Tipping 
+
+Dogecoin’s popularity can be attributed in part to its history as a tipping currency. In the early days, Dogecoin was widely used as a tipping currency on platforms like Reddit, and since Dogecoin’s insurgence in popularity in 2021, using Dogecoin a tipping currency is now back in vogue! Shibes show their appreciation to fellow Shibes on social-media platforms like Reddit by tipping them Dogecoin using popular bots, such as SoDogeTip.  

```

Comment:
```suggestion
Dogecoin’s popularity can be attributed in part to its history as a tipping currency. In the early days, Dogecoin was widely used as a tipping currency on platforms like Reddit, and since Dogecoin’s insurgence in popularity in 2021, using Dogecoin as a tipping currency is now back in vogue! Shibes show their appreciation to fellow Shibes on social-media platforms like Reddit by tipping them Dogecoin using popular bots, such as SoDogeTip.  

```

Reply:
Relative paths are better as they allow testing also locally.